pkg/proto/nbcontract/v1/customlinuxosconfig.proto

```diff
@@ -46,14 +46,15 @@ message SysctlConfig {
   optional int32 NetIpv4NeighDefaultGcThresh3 = 18;
   optional int32 NetNetfilterNfConntrackMax = 19;
   optional int32 NetNetfilterNfConntrackBuckets = 20;
-  optional int32 FsInotifyMaxUserWatches = 21;
-  optional int32 FsFileMax = 22;
-  optional int32 FsAioMaxNr = 23;
-  optional int32 FsNrOpen = 24;
-  optional int32 KernelThreadsMax = 25;
-  optional int32 VMMaxMapCount = 26;
-  optional int32 VMSwappiness = 27;
-  optional int32 VMVfsCachePressure = 28;
+  optional int32 FsInotifyMaxUserInstances = 21;
```

Comment:
Is this proto contract in use currently? If yes, then reassigning fields like this will break the backward compatibility. To avoid, this field should be added as 

`optional int32 FsInotifyMaxUserInstances = 29;`

and everything else left as is. However, if the files are not in use - that's safe to do (for now)

Reply:
more context: https://protobuf.dev/programming-guides/proto3/#assigning